### ISSUE TITLE
Bug: Notification sound is ringing while the desktop notification is not showing

### ISSUE THREAD

**Issue by knodelle** [tracim/tracim#5455]
## Description and expectations

The call sound is sometimes on while the notifications are not showing, so the user only hears a sound but they don't know where it is coming from unless they go to the right tracim tab.

### How to reproduce

1. Open several tabs (the more the better)
2. Go on another app, outside of the browser 
3. Make sure your notifications are not enabled from the tracim instance that is calling you

For more information on how to reproduce, ask @PhilippeAccorsi 

### Version information

- Tracim version: [e.g. v2.7, Mon Tracim]
- Web browser: [e.g. Google Chrome, Firefox, Safari]
- Version: [e.g. 22, latest]
- Device / System: [e.g. PC, iPhone 6]

### Screenshots

________________________________________________

## Prerequisites

~~PR: #5461~~
PR: #5467

**Comment by knodelle** [comment]
After an internal discussion about how to handle this issue, we decided to keep a ringing sound even when the notification are not authorized. And we still want to warn the user that someone is calling, even if they are not on the Tracim app.

Instead, we decided to ask the user for notifications when they connect on Tracim. This way, there is less chance for them to have someone calling without having just the sound.
